<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewStudent</class>
 <widget class="QDialog" name="NewStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ثبت دانشجوی جدید</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="font">
         <font>
          <family>Abyssinica SIL</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <property name="tabSpacing">
         <number>6</number>
        </property>
        <widget class="QWidget" name="personInfo_toolBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>615</width>
           <height>316</height>
          </rect>
         </property>
         <attribute name="label">
          <string notr="true">اطلاعات شخصی</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="name_led">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>نام</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="family_led">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>5</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>نام خانوادگی</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fatherName_led">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>5</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>نام پدر</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="melliCode_led">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>5</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>کد ملی</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="shsh_led">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>5</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>شماره شناسنامه</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stuInfo_toolBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>615</width>
           <height>316</height>
          </rect>
         </property>
         <attribute name="label">
          <string notr="true">اطلاعات دانشجویی</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="id_led">
            <property name="placeholderText">
             <string>شماره دانشجویی</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="uniName_led">
            <property name="placeholderText">
             <string>نام دانشگاه</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="degree_led">
            <property name="placeholderText">
             <string>مقطع تحصیلی</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="major_led">
            <property name="placeholderText">
             <string>رشته تحصیلی</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="area_led">
            <property name="placeholderText">
             <string>گرایش</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="year_of_entry_led">
            <property name="placeholderText">
             <string>سال ورود</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="termInfo_toolBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-37</y>
           <width>600</width>
           <height>353</height>
          </rect>
         </property>
         <attribute name="label">
          <string>اطلاعات ترم</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="condition_combo">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="locale">
             <locale language="Persian" country="Iran"/>
            </property>
            <property name="currentText">
             <string>وضعیت تحصیلی</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="maxVisibleItems">
             <number>4</number>
            </property>
            <property name="maxCount">
             <number>10</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtBottom</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>وضعیت تحصیلی</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>فعال</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>مرخصی</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>مهمان</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="guestFrom_led">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>دانشجوی مهمان از</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="guestTo_led">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="placeholderText">
             <string>دانشجوی مهمان به</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="halfYear_combo">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="locale">
             <locale language="Persian" country="Iran"/>
            </property>
            <property name="currentText">
             <string>نیم سال</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="maxVisibleItems">
             <number>4</number>
            </property>
            <property name="maxCount">
             <number>4</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtBottom</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">نیم سال</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>اول</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>دوم</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>تابستانه</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yearNum_spinBox">
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="prefix">
             <string notr="true">سال تحصیلی </string>
            </property>
            <property name="minimum">
             <number>80</number>
            </property>
            <property name="maximum">
             <number>99</number>
            </property>
            <property name="value">
             <number>94</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="termNum_spinBox">
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="prefix">
             <string>شماره ترم </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="unitNum_spinBox">
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="prefix">
             <string notr="true">تعداد واحد اخذ شده </string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>24</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lessonNum_spinBox">
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="prefix">
             <string notr="true">تعداد دروس </string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>24</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="lesson_table">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideMiddle</enum>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>199</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>نام درس</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>نمره</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>تعداد واحد</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_led</tabstop>
  <tabstop>family_led</tabstop>
  <tabstop>fatherName_led</tabstop>
  <tabstop>melliCode_led</tabstop>
  <tabstop>shsh_led</tabstop>
  <tabstop>id_led</tabstop>
  <tabstop>uniName_led</tabstop>
  <tabstop>degree_led</tabstop>
  <tabstop>major_led</tabstop>
  <tabstop>area_led</tabstop>
  <tabstop>year_of_entry_led</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
